source/assets/scripts/main.js

```diff
@@ -203,6 +214,77 @@ async function fetchRandomAPI() {
  *                      RECIPE CARDS
  ****************************************************************************/
 
+/**
+ * function binds the Featured Recipe Cards to the homepage
```

Comment:
Soon, we'll have to change this comment to be compatible with automatic documenting with `JSDoc`

Reply:
There aren't any parameters or return types. Does it need anything else to be JSDoc compatible?

Reply:
Wait a second, you totally right. My bad~